test_data/third_party/2and3/six_test.py

```diff
@@ -0,0 +1,17 @@
+import pip  # type: ignore
+
+pip.main('install six==1.10.0'.split())
```

Comment:
I wonder if it would be better to have the requirements in a text file, similar to how other dependencies are dealt with (e.g. `requirements-tests-py3.txt`)?

Also, maybe we should require that we explicitly define the versions of all packages being installed, including those that can be automatically installed by pip through dependencies. Otherwise, an updated package on PyPI may break our tests.